graylog2-web-interface/src/components/search/MessageFieldDescription.jsx

```diff
@@ -50,6 +50,15 @@ class MessageFieldDescription extends React.Component {
     SearchStore.addSearchTerm(this.props.fieldName, this.props.fieldValue);
   };
 
+  _getFieldValue() {
+    if (this.props.isDecorated && ('type' in this.props.fieldValue) && (this.props.fieldValue.type === 'a')) {
+      const link = this.props.fieldValue.href;
+      return React.createElement('a', {href: link}, link);
+    } else {
```

Comment:
The `else` case is not really needed (since we return in the `if` branch) and can be deleted.

Reply:
Fixed all the requested changes.